src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/domain/ccd/message/EventInformation.java

```diff
@@ -7,15 +7,13 @@
 import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
 import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
 import lombok.Builder;
-import lombok.EqualsAndHashCode;
 import lombok.ToString;
 
 import java.time.LocalDateTime;
 import java.util.Locale;
 import javax.validation.constraints.NotEmpty;
 
 @ToString
-@EqualsAndHashCode
```

Comment:
Why are we removing this annotation? This is usually a good thing in the Pojo to ensure uniqueness